Src/control.c

```diff
@@ -127,34 +115,34 @@ void PWM_ISR_CH2_Callback() {
 }
 
 // SysTick executes once each ms
-void PWM_SysTick_Callback() {
+void PWM_SysTick_Callback(void) {
   pwm_timeout++;
   // Stop after 500 ms without PPM signal
   if(pwm_timeout > 500) {
     //pwm_captured_ch1_value = 500;
     pwm_captured_ch2_value = 500;
-    pwm_timeout = 0;
+    pwm_timeout = 500;
```

Comment:
This acts as a reset of the PWM timeout, allowing it to retry after the safe values have been set. It is supposed to be 0.

Reply:
Ok.